MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.h

```diff
@@ -24,8 +24,8 @@ typedef NS_ENUM(NSUInteger, MUXSDKPlayerState) {
 
 
 @interface MUXSDKPlayerBinding : NSObject {
-@private
     NSString *_name;
+@private
```

Comment:
why did this move down?

Reply:
oh ya... that was me trying to expose the instance variable for debugging purposes